### PR TITLE
Refactor/error handling

### LINKED ISSUE #99: Error handling for search feature
Add a "Sorry, no results" error

### PR DESCRIPTION
**Pull request by jfogiato**
Files Changed: scripts.js

Change Summary: This PR adds error handling to the search function in the case where no recipes match the users' search.

Notes: Verbiage is subject to change, but I had fun with it :) 

Closes #99

### CODE REVIEW

**Review by LordSchwifty — APPROVED** [review]
Nice job error handling. I think the error message looks good!

**Review by jsahim — APPROVED** [review]
Great solution!

File: src/scripts.js [hunk 1]
```diff
@@ -121,8 +121,13 @@ function filterRecipes(event) {
 function searchRecipes() {
   let keyword = searchBar.value;
   let searchedRecipes = savedView ? currentUser.filterSavedByName(keyword) : recipeRepo.filterByName(keyword);
-  currentView = searchedRecipes;
-  createRecipeCards(currentView);
+  if (searchedRecipes.length) {
```
Comment (qrispi) [inline src/scripts.js:124]: Very clever! Love this solution

**Review by qrispi — APPROVED** [review]
Great fix Joe!

**Pull request merged**